src/omv/sensors/genx320.c

```diff
@@ -166,7 +170,7 @@ static int reset(omv_csi_t *csi) {
     }
 
     if (psee_ehc_activate(&psee_ehc, EHC_ALGO_DIFF3D, 0, EHC_DIFF3D_N_BITS_SIZE,
-                          EVT_SCALE_PERIOD(INTEGRATION_DEF_PREIOD), EHC_WITHOUT_PADDING) != EHC_OK) {
+                          INTEGRATION_DEF_PREIOD, EHC_WITHOUT_PADDING) != EHC_OK) {
```

Comment:
The new ISSD internally makes the camera timebase not need scaling by the clock anymore.